### PR TITLE
Update pre-industrial configuration with latest payu updates

### PR DESCRIPTION
**Pull request by blimlim**
This pull request completes the pre-industrial half of #40. It modifies the configuration with the following changes:

- Remove um_env.py file now that it has been replaced by the um_env.yaml file.
-  Remove unused atmosphere configuration files #27.
- Add/update the following settings in the config.yaml file:
    - Add commented out `shortpath` and `project` settings
    - Add sync settings with default false setting. Add comment explaining compatibility with postscripts
    - Add restart_freq setting with default value of 5.
    - Add runlog and platform settings with default values.
    - Add ` manifest:  reproduce:    exe: True` as in the OM2 configurations.

It would be great to get a check over these changes, especially to confirm that the default values for several of the new `config.yaml` settings are suitable for ESM1.5.

### CODE REVIEW

**Comment by blimlim** [comment]
I'll wait until #38 is merged and update this branch with the changes before trying to merge.

File: config.yaml
Comment (aidanheerdegen) [inline config.yaml:82]: The default is 1, so we could remove this to keep things as simple as possible.

We also have the option of making this a larger number. The [queue limits for `gadi`](https://opus.nci.org.au/display/Help/Queue+Limits) are such that we could run this for 48 hours per submit. This probably equates to 20 year runs per submit (assuming there is a little padding in the 2.5 hour queue limit).

This would require setting `walltime: 48:00:00`.

The downside is when testing and running less than 20 hours it would potentially increase the queueing time because it is asking for more resources (`walltime`).

I've [made an issue to look into changing the default behaviour](https://github.com/payu-org/payu/issues/479), but that won't be done before we release, so we would have to put up with the sub-optimal behaviour if we increased this.

File: config.yaml
Comment (aidanheerdegen) [inline config.yaml:12]: Do we want to remove this? 

It used to be handy to reduce the possibility of "namespace collision" between other `access` models, but this is no longer an issue with the experiment ID changes, so perhaps it is ok.

**Review by aidanheerdegen — CHANGES_REQUESTED** [review]
Some changes, some comments and some questions.

File: config.yaml
Comment (blimlim) [reply config.yaml:65]: That looks great, just added it in.

File: config.yaml
Comment (blimlim) [reply config.yaml:12]: I've kept the laboratory setting in a bit higher up, but just had updated to comment to be accurate now that the `shortpath` setting has been added in:
```
# Note: if laboratory is relative path, it is relative to shortpath/$USER
laboratory: access-esm
```

File: config.yaml
Comment (blimlim) [reply config.yaml:2]: That's a good idea. Would it be better to link to just the Hive docs, or both the Hive docs and the payu config documentation which goes into more detail?

File: config.yaml
Comment (blimlim) [reply config.yaml:82]: Good idea. I'd kept it in originally as the Hive docs currently use it when discussing [ways to change the total simulation lengths](https://access-hive.org.au/models/run-a-model/run-access-esm/#understand-runtime-runspersub-and-n-parameters). I guess there's no reason to have the current Hive docs as the controlling factor. 

With the proposed changes it sounds like we would have a large value like 20 as the default – Do you think it makes sense to set this now (and increase the walltime) so that there are minimal changes when we add the future updates?

File: config.yaml
Comment (blimlim) [reply config.yaml:143]: I've just checked with @MartinDix. For the CMIP6 runs, restarts were kept every 10 years, and apparently this was pretty reasonable in terms of the storage space. I'll add this as the default.

File: config.yaml
Comment (aidanheerdegen) [reply config.yaml]: Both I think.

Comment (blimlim) [reply config.yaml]: Ok great! I've added them both in

File: config.yaml
Comment (aidanheerdegen) [reply config.yaml]: Ok, scratch this (my) idea. 

We should wait until we have a better solution implemented in `payu`. It's nice and simple if it's just a single run, and if we get complaints about poor throughput we can suggest this as a solution.

Sorry! I know you've made that change already.

Comment (blimlim) [reply config.yaml:135]: No worries! I've just switched it back to 1 and reduced the walltime back to 2:30:00

File: config.yaml
Comment (aidanheerdegen) [inline config.yaml:28]: @jo-basevi has just deployed a version of `payu` that supports the `module use/load` features. When this version of `payu` is used it throws an error that there were multiple versions of models available because `module use /g/data/vk83/modules` also picks up the modules in the `access-models` subdirectory.

The solution is to change to
```yaml
modules:
  use:
      - /g/data/vk83/modules/
  load:
      - access-esm1p5/2024.05.0
```
which should be a no-op if `/g/data/vk83/modules/` is already `use`'d (I just tested outside of `payu` and it works fine, but please do check).

**Review by aidanheerdegen — CHANGES_REQUESTED** [review]
Sorry! One more change required it seems.

File: config.yaml
Comment (blimlim) [reply config.yaml:28]: Just added this in and running the simulation with these changes is working!

**Review by aidanheerdegen — APPROVED** [review]
All good to merge, thanks @blimlim!

**Comment by blimlim** [comment]
Thanks for the review! There will be a similar one soon with the same changes for the `historical` configuration. I've added in all the changes from the discussion here and so hopefully that one won't be too involved!

**Pull request merged**
